spec/spec_helper.rb

```diff
@@ -38,6 +39,7 @@
 Sidekiq::Logging.logger = nil
 
 require 'webmock/rspec'
+require 'climate_control'
```

Comment:
oops think we've managed to merge in requiring this twice.

Reply:
Done